.github/workflows/infracost.yml

```diff
@@ -140,11 +140,19 @@ jobs:
             echo "No changes detected in infrastructure"
           fi
 
+      # parse pull_request.base.ref before generating artifacts
+      - name: Parse artifact name
+        env:
+          BASE_REF: ${{ github.event.pull_request.base.ref }}
+        id: parse
+        run: |
+          echo "::set-output name=artifact_name_base::$(echo $BASE_REF | sed 's/\//-/g')"
```

Comment:
this syntax is  being deprecated - better to replace now https://docs.github.com/en/actions/using-jobs/defining-outputs-for-jobs

Reply:
oh, thanks for sharing didn't know they updated that